README.md

```diff
@@ -105,7 +105,7 @@ which gets used when you create a `Runner` with no config.
 
 ## Linters
 
-`erb-lint` comes with 2 linters on-board: `DeprecatedClasses` and `FinalNewline`, each with their own linter-specific options.
+`erb-lint` comes with 3 linters on-board: `DeprecatedClasses`, `FinalNewline`, and `ContentStyle`, each with their own linter-specific options.
```

Comment:
ContentStyle does a lot more so I think you should place it before `FinalNewline`.